node/src/chain_spec.rs

```diff
@@ -114,14 +104,14 @@ pub fn development_config(id: ParaId) -> ChainSpec {
         None,
         Some(properties),
         Extensions {
-            relay_chain: "rococo-dev".into(),
+            relay_chain: "rococo-local".into(),
```

Comment:
we're using this for both Rococo and ChaChaCha?

Reply:
I thought it was according to rococo-v1 configuration https://github.com/substrate-developer-hub/substrate-parachain-template/blob/40b858149b212e493da08d80e0fc5b06a6b0b72d/node/src/chain_spec.rs#L70

In Polkadot0.9.9 https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.9/node/src/chain_spec.rs#L76, it is `rococo-local`

Reply:
please let me know if you still want me to rename it

Reply:
it looks like you've already done that here https://github.com/DataHighway-DHX/node/pull/217/files/b7fffa893365f7396722881083541ddf7c3b91c9..5b745704eda73019a834a39d17f6690999097068, so we can close this issue